packages/jellyfish-api-core/__tests__/category/blockchain/getBlockFilter.test.ts

```diff
@@ -0,0 +1,50 @@
+import { MasterNodeRegTestContainer, StartFlags } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Retrieve a BIP 157 content filter for a particular block', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    const startFlags: StartFlags[] = [{ name: 'blockfilterindex', value: 1 }]
+    await container.start({ startFlags: startFlags })
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should retrieve a BIP 157 content filter for genesis block', async () => {
+    {
+      const blockhash = await client.blockchain.getBlockHash(0)
+      const promise = client.blockchain.getBlockFilter(blockhash)
+
+      expect(await promise).toStrictEqual({
+        filter: expect.stringMatching(/^[0-f]{1,}$/),
```

Comment:
```suggestion
        filter: expect.stringMatching(/^[0-9a-f]{1,}$/),
```

To fix due to code scanning alert